backend/alembic/versions/27c6ecc08586_permission_framework.py

```diff
@@ -74,7 +74,7 @@ def upgrade() -> None:
         sa.Column(
             "credential_json",
             postgresql.JSONB(astext_type=sa.Text()),
-            nullable=True,
+            nullable=False,
```

Comment:
This matches the model code. auto migration added this edit in the new migration but it felt better to just update it here.